stdlib/messaging/kafka/src/main/ballerina/src/kafka/consumer.bal

```diff
@@ -263,7 +262,7 @@ public type Consumer client object {
     # + duration - Timeout duration for the get beginning offsets execution.
     # + return - Starting offsets for the given partitions if executes successfully, returns `kafka:ConsumerError` if the operation fails.
     public remote function getBeginningOffsets(TopicPartition[] partitions, public int duration = -1)
-                               returns PartitionOffset[]|ConsumerError {
+                                                returns PartitionOffset[]|ConsumerError {
```

Comment:
Indentation is different from one to another. Shall we unify it to proper standard?